ui/__config__/webpack.config.js

```diff
@@ -44,8 +44,26 @@ const config = {
             },
             {
                 test: /\.(js|jsx)$/,
-                exclude: /(node_modules)/,
-                loader: 'babel-loader',
+                oneOf: [
+                    {
+                        test: /(\/inline)/,
+                        exclude: /node_modules/,
+                        use: [
+                            'raw-loader',
+                            {
+                                loader: 'babel-loader',
+                                options: {
+                                    comments: false,
+                                    babelrc: false,
```

Comment:
As discussed, rather than disabling `.babelrc` to get around the implicit `import h`, can we instead remove `babel-plugin-provide-modules` from `.babelrc`, and then extend the `.babelrc` in`webpack.config.js`, adding the plugin specifically for JSX files (`test: /.jsx$/`)?

I found an example of [extending `.babelrc`](https://medium.com/@jberniertech/upgrading-to-webpack-2-and-tree-shaking-results-e68bd203c974#726a), maybe this will help